test/ch/connection_test.exs

```diff
@@ -523,6 +523,7 @@ defmodule Ch.ConnectionTest do
               }} = Ch.query(conn, "SELECT * FROM t_uuid ORDER BY y")
     end
 
+    @tag :skip
```

Comment:
Latest ClickHouse now has "now JSON" https://clickhouse.com/docs/en/sql-reference/data-types/newjson, I'll add support to it later. For now the test for "old JSON" is just skipped.